assets/prefabs/CookingRecipes.prefab

```diff
@@ -248,6 +248,90 @@
         },
         "outputCount": 1
       }
+      "Cooking:BoiledEgg": {
+        "inputs": {
+          "Cooking:Egg": 1
+        },
+        "outputCount": 1
+      }
+      "Cooking:CookedBean": {
+        "inputs": {
+          "Cooking:bean":1,
+          "Cooking:onion":1,
+          "Cooking:tomato":1,
+          "Cooking:salt":1
+        },
+        "outputCount": 1
+      }
+      "CookedBrownRice": {
```

Comment:
Please add the `Cooking:` prefab to avoid bugs if another module also provides an item with the same name.
```suggestion
      "Cooking:CookedBrownRice": {
```